azkaban-exec-server/src/main/java/azkaban/execapp/FlowRunnerManager.java

```diff
@@ -271,10 +273,15 @@ public void setGlobalProps(Props globalProps) {
     // Every 2 mins clean the recently finished list
     private static final long RECENTLY_FINISHED_INTERVAL_MS = 2 * 60 * 1000;
 
+    // Every 5 mins kill flows running longer than allowed max running time
+    private static final long LONG_RUNNING_FLOW_KILLING_INTERVAL_MS = 5 * 60 * 1000;
```

Comment:
Interesting story. Thanks!

Regarding the suggestion to use Duration, yes, I agree. I saw below in another comment that Cheng agrees too.

My comment was referring to 
" // Max flow running time in mins, server will kill flows running longer than this setting.
 +  // if not set or <= 0, then there's no restriction on running time.
 +  public static final String AZKABAN_MAX_FLOW_RUNNING_MINS = "azkaban.server.max.flow.running.mins";
"
This is a admin facing config variable name. I think having the unit in the name in minutes makes sense. Usage of Duration doesn't seem to apply in this context.